src/block.rs

```diff
@@ -202,14 +202,14 @@ impl Block {
             // and the block id
             let mut current_sid = 0;
 
-            for slip in tx.outputs_mut().iter_mut() {
+            for slip in tx.body.outputs_mut().iter_mut() {
```

Comment:
Small nitpick, but any reason we want accessors on the `body` instead of on `Transaction`?

Reply:
If I'm only dealing with a body(now called UnsignedTransaction), I'd still like to be able to get the outputs etc.

Reply:
we can wrap them if you really want to.

David and you are both pushing back against this "body" concept but that basically means we need to write a serialize() function instead, which in my view defeats the purpose of using bincode in V1, but I might just do it.